### ISSUE TITLE
E3002- SNS Tags Property failing validation

### ISSUE THREAD

**Issue by sliedig** [aws-cloudformation/cfn-python-lint#1173]
*cfn-lint version: (`cfn-lint --version`)* 0.24.5

*Description of issue.*

Incorrectly errors on Tag property for SNS topics - 
`E3002 Invalid Property Resources/<resourceid>/Properties/Tags` 
Tags are a supported property for CFN, specification and documentation needs to be updated.

Example:
``` yaml
  MyTopic:
    Type: AWS::SNS::Topic
    Properties:
      TopicName: MySNSTopic
      Tags:
      - Key: project
        Value: demo
```

**Comment by kddejong** [comment]
Validated this.  Also its not in the CloudFormation spec so we will have to patch it in.

**Comment by kddejong** [comment]
Fixed in master now waiting for the next release.

**Comment by kddejong** [comment]
https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/7
